proto/viam/app/datasync/v1/data_sync.proto

```diff
@@ -71,10 +72,20 @@ message StreamingDataCaptureUploadResponse {
   string file_id = 1;
 }
 
-// SensorMetadata contains the time the sensor data was requested and was received.
+enum MimeType {
+  MIME_TYPE_UNSPECIFIED = 0;
+  MIME_TYPE_IMAGE_JPEG = 1;
+  MIME_TYPE_IMAGE_PNG = 2;
+  MIME_TYPE_PCD = 3;
```

Comment:
optional. Might this be `MIME_TYPE_APPLICATION_PCD`? I see there’s no official type defined but the convention seems to be `application/x-pcd` e.g.
https://wiki.hanzheteng.com/development/linux/tricks

Reply:
good idea, done